java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/IngestBatcherIT.java

```diff
@@ -46,6 +49,12 @@ public class IngestBatcherIT {
     void setUp() {
         sleeper.connectToInstance(MAIN);
         sleeper.ingest().batcher().clearStore();
+        sleeper.ingest().statusStores().clear();
```

Comment:
There's a problem here - if an ingest task is still running, we don't want to delete it. We'd like it to still show up in the reports.